packages/cli/src/commands/generate/sdl/sdl.js

```diff
@@ -234,5 +234,6 @@ export const handler = async ({
     await tasks.run()
   } catch (e) {
     console.log(c.error(e.message))
```

Comment:
Use stderr, instead of stdout.
```suggestion
    console.error(c.error(e.message))
```